sky/shell/platform/mojo/platform_view_mojo.cc

```diff
@@ -57,6 +57,9 @@ PlatformViewMojo::~PlatformViewMojo() {
 void PlatformViewMojo::Init(mojo::Shell* shell) {
   mojo::ConnectToService(shell, "mojo:native_viewport_service", &viewport_);
 
+  // Grab the application connector so that we can connect to services later
+  shell->CreateApplicationConnector(GetProxy(&connector_));
```

Comment:
The cases that omit the mojo:: are relying on ADL (aka koenig lookup).  The type of the parameter is mojo::InterfacePtr<...> so GetProxy(...) should always work.  This is not the most obvious part of C++ but it is part of the language and makes for cleaner syntax so we use it a fair bit in mojo.